src/System.Data.SqlClient/tests/ManualTests/SQL/SqlRandom/SqlRandomTable.cs

```diff
@@ -0,0 +1,822 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.IO;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
+using System.Diagnostics;
+
+namespace System.Data.SqlClient.ManualTesting.Tests
+{
+    /// <summary>
+    /// represents table with random column types and values in it
+    /// </summary>
+    public class SqlRandomTable
```

Comment:
Is this class used anywhere?  I don't find references of the Constructor outside the Create method of this class. 
And I don't see the reference to Create anywhere either.